### ISSUE TITLE
Error while running in macosx

### ISSUE THREAD

**Issue by arcturus** [TheThingSystem/steward#293]
I get the following exception when running on macosx with node v0.10.24:

alert: [steward] exception diagnostic=Cannot find module 'node-blink1'
alert: [steward] exception stack=[{"fileName":"module.js","lineNumber":338,"functionName":"Function.Module._resolveFilename","typeName":"Function","methodName":"Module._resolveFilename","columnNumber":15,"native":false},{"fileName":"module.js","lineNumber":280,"functionName":"Function.Module._load","typeName":"Function","methodName":"Module._load","columnNumber":25,"native":false},{"fileName":"module.js","lineNumber":364,"functionName":"Module.require","typeName":"Module","methodName":"require","columnNumber":17,"native":false},{"fileName":"module.js","lineNumber":380,"functionName":"require","typeName":"Object","methodName":null,"columnNumber":17,"native":false},{"fileName":"/Users/arcturus/dev/git/community/steward/steward/devices/devices-lighting/lighting-blink1-led.js","lineNumber":4,"functionName":"","typeName":"Object","methodName":null,"columnNumber":19,"native":false},{"fileName":"module.js","lineNumber":456,"functionName":"Module._compile","typeName":"Module","methodName":"_compile","columnNumber":26,"native":false},{"fileName":"module.js","lineNumber":474,"functionName":"Object.Module._extensions..js","typeName":"Object","methodName":"Module._extensions..js","columnNumber":10,"native":false},{"fileName":"module.js","lineNumber":356,"functionName":"Module.load","typeName":"Module","methodName":"load","columnNumber":32,"native":false},{"fileName":"module.js","lineNumber":312,"functionName":"Function.Module._load","typeName":"Function","methodName":"Module._load","columnNumber":12,"native":false},{"fileName":"module.js","lineNumber":364,"functionName":"Module.require","typeName":"Module","methodName":"require","columnNumber":17,"native":false}]
uncaught exception: Error: Cannot find module 'node-blink1'
Press ^C to exit or wait 10 seconds to restart

**Comment by arcturus** [comment]
Had to manually add the package node-blink1

**Comment by mrmakeit** [comment]
`npm install` should have installed node-blink1.  Are you using the homebrew version of node?

**Comment by arcturus** [comment]
I'm using node installed via nmv.

**Comment by mrmakeit** [comment]
Can you try installing blink-1 manually, and let me know how steward reacts?

**Comment by arcturus** [comment]
Yep, that was I did (see 2nd commend). Installing manually works ok and let's steward start

**Comment by mrmakeit** [comment]
Your right, my bad.

Can you send me the versions of node and npm you are using, as well as `git rev-parse HEAD`?

**Comment by arcturus** [comment]
Running node 4.2.0
With commit c000d65f1f431727e54bf0959c521ef359323cb6
